### PR TITLE
Fix issue with complex eigenspectrum in LCMV

### PR DESCRIPTION
**Pull request by agramfort**
taking over #4568

### CODE REVIEW

**Comment by agramfort** [comment]
I confirm that on my phantom data this gives good results

File: mne/beamformer/_lcmv.py [hunk 1]
```diff
@@ -815,11 +821,17 @@ def lcmv_epochs(epochs, forward, noise_cov, data_cov, reg=0.05, label=None,
...
+    .. versionadded:: 0.15.0
+
     reduce_rank : bool
         If True, the rank of the leadfield will be reduced by 1 for each
         spatial location. Setting reduce_rank to True is typically necessary
         if you use a sphere model for MEG as in this case the actual
         rank is 2 not 3.
+
+    .. versionadded:: 0.15.0
+
```
Comment (larsoner) [inline mne/beamformer/_lcmv.py]: All of these are indented too much. And also you only need the newline before, not this extra one after

Comment (larsoner) [reply mne/beamformer/_lcmv.py]: Sorry, indented too little

**Comment by larsoner** [comment]
Travis has a few failures

**Comment by agramfort** [comment]
looks like we can release now !

**Comment by larsoner** [comment]
I think so. Monday? Or should we wait a few days?

BTW CircleCI is broken by some Mayavi update.

**Comment by agramfort** [comment]
let's draft the announcement email and push this asap but after circleci is
fixed so we can update the web site ...

File: mne/beamformer/_lcmv.py [hunk 2]
```diff
@@ -256,21 +255,15 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
...
-            rank_G = reduce_rank
-
-        if rank_G <= 0:
-            raise ValueError('Cannot reduce rank of leadfield to %i.' % rank_G)
-    else:
-        rank_G = estimate_rank(G[:, 0:3], tol='auto')
+            raise NotImplementedError('The computation of spatial filters '
+                                      'with rank reduction using reduce_rank '
+                                      'parameter is not yet implemented with '
+                                      'pick_ori=="max-power".')
```
Comment (britta-wstnr) [inline mne/beamformer/_lcmv.py]: @agramfort it is _only_ implemented with max-power orientation, isn't it?

**Comment by agramfort** [comment]
good catch

fixed 9137358da

**Pull request merged**